Topics/Professionalism.md

```diff
@@ -1,75 +1,155 @@
+
+5. **Avoid technical jargon when possible.** Certain technical jargon may confuse or disengage non-technical members from your presentation. Try to say exactly what you mean instead of using shorthands that are familiar to only other software engineers. This will ensure all audience members benefit in the same way from your presentation.
+
+6. **Focus on impact when explaining technical concepts**. The most important part of your presentation to the non-technical audience members will always be the impact of your work on the business and on the consumers/customers. To them, it does not matter what specific technology was used to decrease the CPU time of the product by 50%. It only matters that the consumer/customer can now enjoy a user experience that is 50% faster or that the business can now make 50% more money. It is not their job to worry about the technical details behind the product, so it is often not relevant to share those details with non-technical stakeholders or co-workers. "Focus on the initiatives and pain points that your audience cares most about, and your interactions will have a much greater impact with executives and other non-technical employees at your organization."
+
+7. **Encourage questions**. Sometimes audience members may be uncomfortable speaking up and asking questions when something is not clear to them during your presentation. "As you go through your presentation, pause and ask, 'Are there any questions?' Create an atmosphere where stakeholders feel comfortable asking for clarification and starting conversations. "
+
+# External Resources for Professional Development
+
+Included in this section is a list of resources of various types that provide interesting insights into the worlds of professionalism and professional development.
+
+### Articles:
+
+1. https://ethics.acm.org/code-of-ethics/software-engineering-code/ ----- "Teaching and Learning Guide for: Software Engineering Ethics" by Don Gotterbarn, ACM SIGCAS Computers and Society
+
+2. https://medium.com/geekculture/soft-skills-for-software-engineers-part-2-get-things-done-cb41a49b9958 ---- Soft Skills for Software Engineers — Part 2: Get Things Done by Andrei Vlasiuk
+   
+3. https://medium.com/geekculture/soft-skills-for-software-engineers-part-3-effective-mindset-d9072d6ae7f9 ------ Soft Skills for Software Engineers — Part 3: Effective Mindset by Andrei Vlasiuk
+
+### Blogs:
+https://www.pluralsight.com/blog/it-ops/training-tips-for-it-pros ----- 4 reasons IT pros need to train (even when they think they don't) By Don Jones on August 17, 2015
+
+### Podcasts:
+https://www.podchaser.com/podcasts/software-engineering-daily-62587 ----- Software Engineering Daily - Daily interviews about technical software topics.
+
+### School Resources:
```

Comment:
As a style suggestion, I think changing this title to "Resources from UofT" or any similar title would be more beneficial.

Reply:
That makes sense. I also think that would be a better title. Thank you for the suggestion!